scripts/variantstore/wdl/extract/import_gvs.py

```diff
@@ -407,8 +419,6 @@ def convert_array_with_id_keys_to_dense_array(arr, ids, drop=[]):
         vd = vd.annotate_entries(FT=~ft.any_no & (ft.any_yes | ((~ft.any_snp | ft.any_snp_ok) & (~ft.any_indel | ft.any_indel_ok))))
 
         vd = vd.drop('allele_NO', 'allele_YES', 'allele_is_snp', 'allele_OK')
+        vd = vd.rename({'as_vqsr': 'as_vets'})
 
-        hl.vds.VariantDataset(
-            reference_data=rd,
-            variant_data=vd,
-        ).write(final_path, overwrite=True)
+        vd.write(os.path.join(final_path, 'variant_data'), overwrite=True)
```

Comment:
if my understanding of the above few lines is correct, here we are _only_ writing the variant_data matrix table and not the reference_data matrix table that we have starting on line 411. 
I'm not sure of the best version of this going forward (do we build in a flag that assumes the worst? do we create a check that assumes the worst may have occurred) but this doesn't feel right as is

Reply:
Corrected in code